frontend-react/src/pages/Submissions.tsx

```diff
@@ -0,0 +1,103 @@
+import { GLOBAL_STORAGE_KEYS } from "../components/GlobalContextProvider";
+
+import { ErrorPage } from "./error/ErrorPage";
+
+const OrgName = () => {
+    const orgName: string = useOrgName();
+    return (
+        <span id="orgName" className="text-normal text-base">
+            {orgName}
+        </span>
+    );
+};
+
+function Submissions() {
+    const { authState } = useOktaAuth();
+    const organization = localStorage.getItem(GLOBAL_STORAGE_KEYS.GLOBAL_ORG);
+    const [submissions, setSubmissions] = useState([]);
+
+    fetch(
+        `${process.env.REACT_APP_BACKEND_URL}/api/history/${organization}/submissions`,
+        {
+            headers: {
+                Authorization: `Bearer ${authState?.accessToken?.accessToken}`,
+            },
+        }
+    )
+        .then((res) => res.json())
+        .then((submissionJson) => {
+            setSubmissions(JSON.parse(submissionJson));
+        });
+
+    return (
+        <NetworkErrorBoundary
+            fallbackComponent={() => <ErrorPage type="page" />}
+        >
+            <Helmet>
+                <title>Submissions | {process.env.REACT_APP_TITLE}</title>
+            </Helmet>
+            <section className="grid-container margin-top-5">
+                <h3 className="margin-bottom-0">
+                    <Suspense
+                        fallback={
+                            <span className="text-normal text-base">
+                                Loading Info...
+                            </span>
+                        }
+                    >
+                        <OrgName />
+                    </Suspense>
+                </h3>
+                <h1 className="margin-top-0 margin-bottom-0">COVID-19</h1>
+            </section>
+            <div className="grid-container usa-section margin-bottom-10">
+                <div className="grid-col-12">
+                    <h2>Submissions</h2>
+                    <table
+                        className="usa-table usa-table--borderless prime-table"
+                        aria-label="Submission history from the last 30 days"
+                    >
+                        <thead>
+                            <tr>
+                                <th scope="col">Date/time submitted</th>
+                                <th scope="col">File</th>
+                                <th scope="col">Records</th>
+                                <th scope="col">Report ID</th>
+                                <th scope="col">Warnings</th>
+                            </tr>
+                        </thead>
+                        <tbody id="tBody" className="font-mono-2xs">
+                            {submissions.map((s, i) => {
+                                return (
+                                    <tr key={"submission_" + i}>
+                                        <th scope="row">
+                                            {moment
+                                                .utc(s["createdAt"])
```

Comment:
I have a general dislike of the `moment` library. It can be a bit heavy weight for what it does.

For example, this can be done like this with standard javascript:
```javascript
console.log(new Intl.DateTimeFormat('en-US', {
  year: 'numeric', month: 'numeric', day: 'numeric',
  hour: 'numeric', minute: 'numeric', second: 'numeric',
}).format(new Date())); // 12/14/2021, 7:56:30 AM

```

But it's fine to leave it, just adding this comment for awareness.

Reply:
I like this. I used moment since that is what we use on the /daily-data page. I'll create a ticket to clean up uses of moment